pipeline/builder/builder.go

```diff
@@ -255,6 +267,39 @@ func (b *Builder) buildDeployEmbeddedManifestStage(index int, s config.Stage) (*
 		ds.Manifests = append(ds.Manifests, objs...)
 	}
 
+	// Generate the configurator config map
+	for _, configuratorFile := range maniStage.ConfiguratorFiles {
+
+		file, err := ioutil.ReadFile(configuratorFile.File)
+		if err != nil {
+			return nil, err
+		}
+
+		env := "default"
```

Comment:
the default environment should be the environment the stage's account is?

Reply:
Set to Stages[s.Account] by default
value will be overridden if set in the yml file
as a last resort, we will fall back to using 'default'